code/modules/events/meateor_wave.dm

```diff
@@ -5,7 +5,7 @@
 	max_occurrences = 1
 
 /datum/round_event/meteor_wave/meaty/announce()
-	priority_announce("Meaty ores have been detected on collision course with the station.", "Oh Crap, Get The Mop.",'sound/AI/meteors.ogg')
+	priority_announce("Meaty ores have been detected on collision course with the station.", "Oh crap, get the mop.",'sound/AI/meteors.ogg')
```

Comment:
This is actually inconsistent with other announcements so it'll look weird and somebody will revert it a few months later.